common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandResult.java

```diff
@@ -36,6 +44,12 @@ public String getCorrelationId() {
     private final Object mResult;
     private final String mCorrelationId;
 
+    @Nullable
+    @Setter
+    @Getter
```

Comment:
Do we want setters and getters on this value?

Reply:
Yes, getting used in CommandDispatcher.java